.github/workflows/build-publish.yaml

```diff
@@ -19,13 +23,13 @@ jobs:
         run: |
           IMAGE_VERSION=$(cat VERSION)
           SHORT_SHA1=$(git rev-parse --short HEAD)
+          PLATFORMS=$(cat PLATFORMS)
           echo "Building version ${IMAGE_VERSION} of image ${IMAGE_NAME}"
-          docker build -t ${IMAGE_NAME} .
-          docker tag "${IMAGE_NAME}" "${IMAGE_NAME}:${IMAGE_VERSION}"
-          echo "$Pushing ${IMAGE_NAME}:${IMAGE_VERSION}"
-          docker push "${IMAGE_NAME}:${IMAGE_VERSION}"
-          docker tag "${IMAGE_NAME}" "${IMAGE_NAME}:${IMAGE_VERSION}-${SHORT_SHA1}"
-          echo "Pushing ${IMAGE_NAME}:${IMAGE_VERSION}-${SHORT_SHA1}"
-          docker push "${IMAGE_NAME}:${IMAGE_VERSION}-${SHORT_SHA1}"
+          docker buildx build --platform ${PLATFORMS} --cache-from=type=registry,ref=${IMAGE_NAME}--cache-to=type=registry,ref=${IMAGE_NAME},mode=max -t ${IMAGE_NAME} .
+          declare -a tags=(${IMAGE_NAME}:${IMAGE_VERSION} ${IMAGE_NAME}:${IMAGE_VERSION}-${SHORT_SHA1})
+          for tag in ${tags[@]}; do
+            echo "Tagging and pushing $tag"
+            docker tag "${IMAGE_NAME}" "$tag"
+            docker push "$tag"
```

Comment:
I think it doesn't work like that, image should be pushed by docker buildx with --push 
And for aliases, need to use docker manifest create or something else
docker tag can only work with a local image which is not multi-arch

Reply:
Given that we will be switching to digests everywhere, do we even care about aliases? Why not just push it as `IMAGE_VERSION-SHORT_SHA1`?

Reply:
yes I'm fine